diff --git a/node_modules/cypress/lib/exec/spawn.js b/node_modules/cypress/lib/exec/spawn.js
index 2b9da0b..3b93acc 100644
--- a/node_modules/cypress/lib/exec/spawn.js
+++ b/node_modules/cypress/lib/exec/spawn.js
@@ -109,9 +109,9 @@ module.exports = {
         var electronArgs = _.clone(args);
         var node11WindowsFix = isPlatform('win32');
 
-        if (verify.needsSandbox()) {
-          electronArgs.push('--no-sandbox');
-        }
+        debug('electron needs sandbox?', verify.needsSandbox())
+        // push no-sandbox all the time
+        electronArgs.push('--no-sandbox');
 
         // strip dev out of child process options
         var stdioOptions = _.pick(options, 'env', 'detached', 'stdio');
diff --git a/node_modules/cypress/lib/tasks/verify.js b/node_modules/cypress/lib/tasks/verify.js
index 7dfbe4c..9bb88b6 100644
--- a/node_modules/cypress/lib/tasks/verify.js
+++ b/node_modules/cypress/lib/tasks/verify.js
@@ -90,10 +90,9 @@ var runSmokeTest = function runSmokeTest(binaryDir, options) {
     var random = _.random(0, 1000);
     var args = ['--smoke-test', '--ping=' + random];
 
-    if (needsSandbox()) {
-      // electron requires --no-sandbox to run as root
-      args.unshift('--no-sandbox');
-    }
+    debug('electron needs sandbox?', needsSandbox())
+    // push no-sandbox all the time
+    args.unshift('--no-sandbox');
 
     if (options.dev) {
       executable = 'node';
